lib/server.js

```diff
@@ -253,24 +264,26 @@ module.exports = function createServer(config, log) {
               blockReason = blockReasons.IP_BAD_REPUTATION
             }
 
+            const result = {
```

Comment:
Any reason for moving this up?

I suppose a pedant could point out that before, a JSObject is only created on success, but I don't think it matters (this isn't a tight loop or anything, so shut up Mr Pedant).